utils/converters/fbx/convert_to_threejs.py

```diff
@@ -305,153 +317,163 @@ def generate_texture_bindings(material_property, material_params):
-
+        material_type = 'MeshLambertMaterial'
+        material_object = {
+          'color': diffuse,
+          'ambient': ambient,
+          'emissive': emissive,
+          'reflectivity': reflectivity,
+          'transparent': transparent,
+          'opacity': opacity
         }
 
     elif material.GetClassId().Is(FbxSurfacePhong.ClassId):
 
-        ambient   = getHex(material.Ambient.Get())
-        diffuse   = getHex(material.Diffuse.Get())
-        emissive  = getHex(material.Emissive.Get())
-        specular  = getHex(material.Specular.Get())
+        ambient   = pack_color(material.Ambient.Get())
+        diffuse   = pack_color(material.Diffuse.Get())
+        emissive  = pack_color(material.Emissive.Get())
+        specular  = pack_color(material.Specular.Get())
         opacity   = 1.0 - material.TransparencyFactor.Get()
         opacity   = 1.0 if opacity == 0 else opacity
-        opacity   = opacity
         shininess = material.Shininess.Get()
         transparent = False
         reflectivity = 1
         bumpScale = 1
 
         material_type = 'MeshPhongMaterial'
-        material_params = {
-
-          'color' : diffuse,
-          'ambient' : ambient,
-          'emissive' : emissive,
-          'specular' : specular,
-          'shininess' : shininess,
-          'bumpScale' : bumpScale,
-          'reflectivity' : reflectivity,
-          'transparent' : transparent,
-          'opacity' : opacity
-
+        material_object = {
+          'color': diffuse,
+          'ambient': ambient,
+          'emissive': emissive,
+          'specular': specular,
+          'shininess': shininess,
+          'bumpScale': bumpScale,
+          'reflectivity': reflectivity,
+          'transparent': transparent,
+          'opacity': opacity
         }
 
     else:
-        print "Unknown type of Material", getMaterialName(material)
+        sys.stderr.write("Unknown type of material: {0}\n".format(get_material_name(material)))
 
-    # default to Lambert Material if the current Material type cannot be handeled
-    if not material_type:
-        ambient   = getHex((0,0,0))
-        diffuse   = getHex((0.5,0.5,0.5))
-        emissive  = getHex((0,0,0))
+    # default to MeshBasicMaterial if the current material type cannot be handeled
+    if material_type == None or material_object == None:
+        ambient   = pack_color((0,0,0))
+        diffuse   = pack_color((0.5,0.5,0.5))
+        emissive  = pack_color((0,0,0))
         opacity   = 1
         transparent = False
         reflectivity = 1
 
-        material_type = 'MeshLambertMaterial'
-        material_params = {
-
-          'color' : diffuse,
-          'ambient' : ambient,
-          'emissive' : emissive,
-          'reflectivity' : reflectivity,
-          'transparent' : transparent,
-          'opacity' : opacity
-
+        material_type = 'MeshBasicMaterial'
+        material_object = {
+          'color': diffuse,
+          'ambient': ambient,
```

Comment:
1. No. Ambient reflectance is hardwired to match diffuse reflectance, now.

2. Yes. Some users are not happy about it.